### PR TITLE
Kata refactoring

### PR DESCRIPTION
**Pull request by valtair1891**

### CODE REVIEW

File: features/kata/install_uninstall.feature [hunk 1]
```diff
@@ -1,53 +1,12 @@
...
-
-  # @author pruan@redhat.com
-  # @case_id OCP-39499
-  @admin
-  @destructive
-  Scenario: kata operator can be installed via CLI with OLM for OCP>=4.8
-    Given the kata-operator is installed using OLM CLI
+Feature: kata install/uninstall related features
+  Background:
+  Given Valid cluster type for kata tests exists
```
Comment (pruan-rht) [inline features/kata/install_uninstall.feature:3]: @valtair1891 indentation, and should be 
`Given a valid cluster type for kata tests exists`

**Comment by pruan-rht** [comment]
@valtair1891 @cpmeadors before doing more reviews, I would like to understand the rationale behind this refactor.  There already exist steps to do installation of `kata`.  https://github.com/openshift/verification-tests/blob/master/features/step_definitions/kata.rb#L1. This PR seems to just pull out the some of the embedded steps out into its own separate step.  IMO, having one step like

`Given kata container has been installed successfully in the "openshift-sandboxed-containers-operator" project
`
is better than having 3 lines
```
    Given Catalog source "redhat-operators" exists in "openshift-marketplace" namespace
    When I install sandboxed-operator in "openshift-sandboxed-containers-operator" namespace
    Then Operator should be up and running
```

File: features/kata/install_uninstall.feature (hunk 1 above)
Comment (valtair1891) [reply features/kata/install_uninstall.feature]: Done

**Comment by valtair1891** [comment]
> @valtair1891 @cpmeadors before doing more reviews, I would like to understand the rationale behind this refactor. There already exist steps to do installation of `kata`. https://github.com/openshift/verification-tests/blob/master/features/step_definitions/kata.rb#L1. This PR seems to just pull out the some of the embedded steps out into its own separate step. IMO, having one step like
> 
> `Given kata container has been installed successfully in the "openshift-sandboxed-containers-operator" project ` is better than having 3 lines
> 
> ```
>     Given Catalog source "redhat-operators" exists in "openshift-marketplace" namespace
>     When I install sandboxed-operator in "openshift-sandboxed-containers-operator" namespace
>     Then Operator should be up and running
> ```
> 
> The idea behind cucumber steps are to hide the implementation details. If you find the existing steps lacks the support you are looking for, then look into modifying or adding new steps to address the issues.

File: features/step_definitions/kata.rb [hunk 2]
```diff
@@ -255,12 +255,11 @@
...
 
 Given /^I wait until sandboxed operator is ready$/ do
-  timeout = 120
+  timeout = 60
   kata_ns = "openshift-sandboxed-containers-operator"
   expected_status = "Installed"
-  wait_for(timeout) do
-    @operator_status = admin.cli_exec(:get, resource: "operators", o: "jsonpath=''{..status.components.refs[4].conditions[0].type}'", n:kata_ns)[:stdout]
-  end
+  sleep(timeout)
```
Comment (pruan-rht) [inline features/step_definitions/kata.rb:261]: @valtair1891 hard-coded sleep

File: features/step_definitions/kata.rb [hunk 3]
```diff
@@ -227,3 +227,39 @@
     end
   end
 end
+
+Given /^a valid cluster type for kata tests exists$/ do
+  accepted_platforms = ['gcp', 'azure']
+  iaas_type = env.iaas[:type] rescue nil
+  raise "Kata installation only supports GCE platform #{iaas_type} is not a valid cluster type" unless accepted_platforms.include? iaas_type
```
Comment (liangxia) [inline features/step_definitions/kata.rb:234]: The above line show `accepted_platforms` as gcp/azure, while here show GCE, mismatch.

**Pull request closed without merging**